README.md

```diff
@@ -63,11 +65,23 @@ You can also configure an instance of a client directly.
 
 ```ruby
 client = IEX::Api::Client.new(
-  publishable_token: 'token',
+  publishable_token: 'publishable_token',
+  secret_token: 'secret_token',
   endpoint: 'https://sandbox.iexapis.com/v1'
 )
 ```
 
+### Make general requests
+
+Some of the endpoints described in [The IEX Cloud API](https://iexcloud.io/docs/api/) are not implemented yet but you 
+can call those endpoints. To do it just call `get`, `post`, `put`, `delete` methods on the `client` passing `path` 
+and `options` as arguments. To call endpoints that use secret API token call `secret_get`, `secret_post`, `secret_put`, 
+`secret_delete` correspondingly.
+
+```ruby
+client.private_post('ref-data/isin', isin: ['US0378331005']) # [{'exchange' => 'NAS', ..., 'symbol' => 'AAPL'}, {'exchange' => 'ETR', ..., 'symbol' => 'APC-GY']
```

Comment:
private or secret?

Reply:
When I started implementing this PR I used `private` for the method names. But then I decided to rename them as `secret` because they use a secret token. 

Using `client.post('whatever', token: secret_token)` I think would be less convenient for end-user razer then `client.secret_post('whatever')` because you must care about the tokens in your code. So most likely the code will look like `client.post('whatever', token: IEX_CONFIG['secret_token'])` or something like this. So as an end-user I would prefer to be responsible only about API calls in my code and not care about the IEX configuration.

So the question is still opened. What do you prefer? `secret` or `private`?

Reply:
I disagree. 99% of the users will not need to specify anything because they use the implemented API wrappers. The only time you'll want to do a token is when you want to call a method that has not been implemented in the library, in which case the implementation detail of specifying a token is OK. I hope you don't feel too strongly about this ;)

Reply:
Furthermore, there's nothing secret or private in those APIs, so I think we shouldn't have methods like this at all. IEX documents the token as "secret" token so that part is fine.

Reply:
Hm. I haven't thought about it in this way. In this case, I leave publishable token as a default option and all API wrappers that need to use the secret token must put it as an argument directly.
What do you think about such implementation?

Reply:
These tokens are unrelated and there's really nothing special about one or the other. Some APIs require the publishable token, and some APIs require the secret token. Tomorrow some other API will require a secret_publishable_token_v2. We should keep things simple.

I vote to modify all the calling code to explicitly use the token desired in the call. This simplifies the need to handle any edge cases of where one is blank, the other one is not. For the user it will make no difference, but you'll have to touch more files.